Add a new important part (#2229)
SEKOIA-IO/documentation: docs/integration/categories/applicative/google_reports.md

    * Type the `an Google workspace admin email`.
  
!!! Important
    * This Google workspace admin email is any user part of the domain **that has** the right to view de Data of Google Workspace

>     * If you are uncertain whether to use a super admin or admin email, ensure that you possess the appropriate permissions in the email for the service you are requesting. For example, if you need to access logs on Google Vault, you need the Access all logs permission.

Reviewer: ```suggestion
    * If you are uncertain whether to use a super admin or admin email, make sure you have the appropriate permissions in the email for the service you are requesting. For example, if you need to access logs on Google Vault, you will need the Access all logs permission.
```
